CONstitution.xml

```diff
@@ -107,99 +107,100 @@
   <chapter id="📜">
     <title>Motions</title>
     <list id="🤔" join="or">
-      <intro>A motion is a proposed action to be taken by the CONsortium as an entity in theory, and once passed allows members to take action to implement it in practice. This includes but is not limited to:</intro>
+      <intro>A <toa definition="true" ref="motion">motion</toa> is a proposed action to be taken by the CONsortium as an entity in theory, and once passed allows members to take action to implement it in practice. This includes but is not limited to:</intro>
       <item id="⚙️">Making changes to the discord server</item>
       <item id="💸">
         <inlist join="andor">
           <item>Fabricating</item>
           <item>transferring</item>
           <item>destroying</item>
         </inlist>
-        fungibles
+        <toa ref="fungible">fungibles</toa>
       </item>
       <item id="🗣️">Declarations</item>
       <item id="📋">Adopting administrative procedures</item>
       <item id="📝">Amending tomes, including The CONstitution</item>
     </list>
-    <text id="🔢">A motion is first Proposed, then voted on, then possibly Passed. A motion can not have any effect before it is passed.</text>
+    <text id="🔢">A <toa ref="motion">motion</toa> is first <toa definition="true" ref="motion-propose">proposed</toa>, after which it is <toa definition="true" ref="motion-state-pending">pending</toa> during which it can be voted on, then possibly <toa definition="true" ref="motion-pass">passed</toa>. A <toa ref="motion">motion</toa> can not have any effect before it is <toa ref="motion-pass">passed</toa>.</text>
```

Comment:
nit: might make more sense as either:
motion-state-proposed, motion-state-pending, motion-state-passed.
OR
motion-proposed, motion-pending, motion-passed.

but right now motion-state-pending seems to be the odd one out.

Reply:
proposal and passing are instantaneous events, whereas pending is a description of the current state of things